package.json

```diff
@@ -73,12 +74,13 @@
         "unlocalize": "rimraf src/nls && git checkout -- src/nls",
         "postinstall": "grunt install",
         "build": "grunt build-browser",
```

Comment:
`grunt build-browser-dev` is just an augmented form of `grunt-build-browser` and realistically, `grunt build-browser` shouldn't be run at any point, so I would say change it to that.